scripts/commerce.js

```diff
@@ -91,7 +91,7 @@ export const priceFieldsFragment = `fragment priceFields on ProductViewPrice {
 export async function commerceEndpointWithQueryParams() {
   const urlWithQueryParams = new URL(getConfigValue('commerce-endpoint'));
   // Set some query parameters for use as a cache-buster. No other purpose.
-  urlWithQueryParams.searchParams.append('ac-storecode', getConfigValue('headers.cs.Magento-Store-Code'));
+  urlWithQueryParams.searchParams.append('ac-storecode', `${getConfigValue('headers.cs.Magento-Store-Code')}-${getConfigValue('headers.cs.Magento-Store-View-Code')}`);
```

Comment:
Maybe we just change the key from ac-storecode to ‘cb’ or ‘buster’ or something.

also can you update docs too?

Reply:
Would it be a reasonable approach to invalidate the cache whenever there's a change in request headers?

I'm thinking specifically about ACO, which uses channels. It might be more robust to generate a hash from the values in `cs.headers` and use that as the "cache key". That way, the cache is directly tied to the specific combination of headers, ensuring that requests with different headers are properly differentiated.